internal/pkg/mapper/defaultbucket/local_save_test.go

```diff
@@ -97,6 +97,9 @@ func TestDefaultBucketMapper_MapBeforeLocalSaveConfig(t *testing.T) {
 
 	// Invoke
 	recipe := fixtures.NewLocalSaveRecipe(configState2.ConfigManifest, configState2.Local)
+	configFile, err := recipe.Files.ConfigJsonFile()
+	assert.NoError(t, err)
+	configFile.Content = content
```

Comment:
@webrouse Pokusil jsem se takto fixnout ty testy, ale vypadá to že jsem tím rozbil většinu funkcionálních. 😅 Hromada jich teď failuje na `panic: found multiple files with tag "config.json`. Asi tuším proč se to děje, ale netuším jak to fixnout. 😃 Respektive zřejmě jsem to tady vyřešil blbě.  Poradíš mi prosím tě co s tím?
![CleanShot 2021-12-08 at 18 09 07](https://user-images.githubusercontent.com/215660/145252258-613ee955-1c8f-49b7-89b9-6205091a18fb.jpg)

Reply:
vyzera to skor ako moja chyba, ktora sa az teraz prejavila (?), pozriem ...

Reply:
TLDR: tym konfiguraciam z orchestratora staci dat iny tag ... idem na to